packages/frameworks-solid-start/src/handler.ts

```diff
@@ -0,0 +1,132 @@
+import { AuthHandler, type AuthAction, type AuthOptions } from "@auth/core";
+import { Cookie, parseString, splitCookiesString } from "set-cookie-parser";
+import { serialize } from "cookie";
+
+export interface SoliduthOptions extends AuthOptions {
+  /**
+   * Defines the base path for the auth routes.
+   * @default '/api/auth'
+   */
+  prefix?: string;
+}
+
+const actions: AuthAction[] = [
+  "providers",
+  "session",
+  "csrf",
+  "signin",
+  "signout",
+  "callback",
+  "verify-request",
+  "error",
+  "_log",
```

Comment:
```suggestion
  "error"
```

This was removed from core for now, thinking about handling client-side logs differently.